.github/pull_request_template.md

```diff
@@ -16,4 +16,4 @@ B | A
 
 - [ ] ⛑️ Tests are included or are not applicable
 - [ ] 📘 Component documentation has been extended, updated or is not applicable
-- [ ] 2️⃣ Backport bugfixes to `stable8` for maintained Vue 2 version.
```

Comment:
`bugfixes` removed, as we need to have `stable8` forward compatible as much as possible until most of the apps are migrated

Reply:
How is this related? We can have v9 exclusive features as well v8 is then still forward compatible.
Only breaking changes need to have forward replacements.

Reply:
Sorry, I meant, feature parity (where possible).

Reply:
V8 featured should be in v9, but other way for now yes later only if needed by specific apps due to complexity of backporting (later = in a year or so)